src/components/Quiz/Quiz.tsx

```
@@ -325,225 +324,256 @@ const QuizFooter = forwardRef<
+              alt="Simulated Result"
+              className="w-[282px] h-auto object-cover"
+            />
+            <div className="flex flex-col gap-2 text-center">
+              <h2 className="text-text-950 font-bold text-lg">
+                Você concluiu o simulado!
+              </h2>
+              <p className="text-text-500 font-sm">
+                Você acertou{' '}
+                {(() => {
+                  const { getActiveQuiz } = useQuizStore();
+                  const activeQuiz = getActiveQuiz();
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)